xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -136,7 +137,7 @@ public Node getNode() {
 
     /** Returns the cert-providers config map. */
     public Map<String, CertificateProviderInfo> getCertProviders() {
```

Comment:
Add `@Nullable` as well as in the constructor and downstream methods?

Reply:
good point. Will do